### PR TITLE
[PR 2:5] feat: reader and writer components

### LINKED ISSUE #19: feat: Having the CLI tool generate something renderable from the tech radar spec file (e.g. HTML or similar)

### LINKED ISSUE #20: feat: Read specification file from a given repository

### PR DESCRIPTION
**Pull request by Slug-Boi**
This PR includes the reader that takes a CSV file in and puts values into GO structs this allows the writer component to create a basic HTML page that has the blips listed. 
You can use the -file flag to specify a path to a .csv file
Resolves #20 
Resolves #19

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,3 +5,19 @@
 module github.com/Agile-Arch-Angels/decentralized-tech-radar_dev
```
Comment (lasselundstenjensen) [inline go.mod:5]: Module is pointing to your fork. Please change it and update the PR.

File: src/View/html_test.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
+package view
+
+import (
+	"os"
+	"strings"
+	"testing"
+
+	Reader "github.com/Agile-Arch-Angels/decentralized-tech-radar_dev/src/SpecReader"
```
Comment (lasselundstenjensen) [inline src/View/html_test.go:8]: Again, pointing to your fork. Please change it and update your PR.

File: src/View/makeHtml.go [hunk 3]
```diff
@@ -0,0 +1,52 @@
+package view
+
+import (
+	Reader "github.com/Agile-Arch-Angels/decentralized-tech-radar_dev/src/SpecReader"
```
Comment (lasselundstenjensen) [inline src/View/makeHtml.go:4]: Again, this is pointing to your fork. Please change it and update your PR.

File: src/main.go [hunk 4]
```diff
@@ -1,14 +1,22 @@
 package main
 
-import(
+import  (
 	"flag"
-	"fmt"
-	"github.com/Agile-Arch-Angels/decentralized-tech-radar_dev/src/SayHello"
+	"github.com/Agile-Arch-Angels/decentralized-tech-radar_dev/src/SpecReader"
```
Comment (lasselundstenjensen) [inline src/main.go:5]: This dependency is pointing to your fork. Please change it and update the PR.

File: docs/user_docs/temp
Comment (lasselundstenjensen) [inline docs/user_docs/temp:1]: Empty file. Remove it or put a comment in it, if it is supposed to be there.
If you want the folder to remain, use a `.gitkeep` file instead.

File: LICENSE
Comment (lasselundstenjensen) [inline LICENSE:1]: You should not be pushing a change to the license if your fork is up to date. It feels like your fork is behind or out of sync?

File: .gitignore
Comment (lasselundstenjensen) [inline .gitignore:1]: Add `go.work.sum` to the `.gitignore` file.

**Review by lasselundstenjensen — CHANGES_REQUESTED** [review]
Fix the changes proposed in the comments before this PR can be approved.

File: LICENSE
Comment (Slug-Boi) [reply LICENSE:1]: I had synced the fork way back and for some reason I guess the change only got caught on this PR? Well either way it is up to date now and should be good for the future PR's :shrug:

**Comment by zqueamish** [comment]
We should now have resolved all comments. The only thing left is resolving the merge conflicts, which we have chosen to leave alone for now, and wait for further instructions:)

**Comment by zqueamish** [comment]
The sync issues between this branch and the main are too severe, and as such we are not able to merge the two. We have incorporated the changes ((https://github.com/NovoNordisk-OpenSource/decentralized-tech-radar/pull/31#pullrequestreview-1970024636)) in later branches, so they will show up, and can seen through those. As such, we want to request to close this PR:)

**Comment by August-Brandt** [comment]
> Module is pointing to your fork. Please change it and update the PR.  

Everywhere the module is pointing to the fork is being changed in PR #49
<br>

> Empty file. Remove it or put a comment in it, if it is supposed to be there. If you want the folder to remain, use a .gitkeep file instead.

This file is no longer exists because of the sync issue
<br>

> Add go.work.sum to the .gitignore file.  

The `go.work.sum` is bring added to the `.gitignore` in PR #73

**Pull request closed without merging**